queue_job/migrations/10.0.1.0.0/end-migration.py

```diff
@@ -31,6 +32,18 @@ def migrate(cr, version):
             })
     # recompute func_string after other addons have adapted their
     # args/kwargs/record_ids
-    records = QueueJob.search([('state', 'not in', [DONE])])
-    records._recompute_todo(QueueJob._fields['func_string'])
-    records.recompute()
+    records = env["queue.job"].search([])
+    total = len(records)
+    count = 0
+    error = 0
+    for record in records:
+        count += 1
+        if not count % 1000:
+            logger.info("Recomputing func_string of job %s of %s",
+                        count, total)
+        try:
+            record._compute_func_string()
+        except KeyError:  # unknown model or non-compliant arguments
+            error += 1
```

Comment:
please log the error so that I see what's going wrong here

Reply:
It's this one https://github.com/OCA/queue/pull/341#discussion_r636695433 but I've pushed a logging commit.

Reply:
I'm attaching the relevant logging to the ticket.